collectors/python.d.plugin/gearman/gearman.chart.py

```diff
@@ -0,0 +1,157 @@
+# Description: dovecot netdata python.d module
+# Author: Kyle Agronick (agronick)
+# SPDX-License-Identifier: GPL-3.0+
+
+# Gearman Netdata Plugin
+
+from bases.FrameworkServices.SocketService import SocketService
+from copy import deepcopy
+
+update_every = 3
```

Comment:
Why `3`? :thinking:

Reply:
I'll removed it. Its arbitrary. It can be anything.